highfidelity/hifi
# WL 21400 -- Setting that lets you switch from cursor to reticle

#### Testing

* Launch this PR build of Interface.
  - (using stock settings) Verify that the mouse cursor displays as the normal arrow icon.
* Enable the new reticle override in "Settings > General...":
![image](https://user-images.githubusercontent.com/94753/28281267-840c9e24-6af4-11e7-93a4-c90beb6c89e5.png)
 (and hit "Save Changes").
  - Verify that the mouse cursor now displays as a circular icon:
![image](https://user-images.githubusercontent.com/94753/28281833-5dcbef24-6af6-11e7-8ccc-3e332be1c89f.png)
* Restart the application.
  - Verify the mouse cursor is still using a circular icon after restart.
* Disable the reticle override in "Settings > General...".
  - Verify that the mouse cursor displays as the normal arrow icon again.
